### DISCUSSION TITLE
Windows CI builds

### DISCUSSION THREAD

**Discussion by Danack** [ImageMagick/ImageMagick#4440] [Development] [answered]
Hi, 

Currently it seems that building ImageMagick from source on Windows requires interacting with a GUI wizard, which is hard to do in a CI environment. 

Is there an alternative way to build and run the configure program on Windows, so that builds can be built automatically without human interaction?

cheers
Dan
Ack

**Comment by dlemstra** [comment, marked as answer]
The `configure.exe` utility can also be run on the command line. You can see an example in our GitHub actions build: https://github.com/ImageMagick/ImageMagick/blob/18e15da2dc6ceae05e95077f85e9d67dac31c5d8/.github/workflows/release.yml#L212.

**Reply by Danack** [reply]
Awesome, thank-you.
